### ISSUE TITLE
Support Platform.sh

### ISSUE THREAD

**Issue by davereid** [davereid/drupal-environment#2]

**Comment by UltraBob** [comment]
Are there any plans to review AaronFeledy's PR for this?

**Comment by AaronFeledy** [comment]
@UltraBob If Dave doesn't have any viable test projects running on Platform at the moment, he may not be able to test the PR himself. I'd say test it if you haven't already and comment your findings on the #4 thread. Community feedback may help move it forward.

**Comment by UltraBob** [comment]
Fair point.  We ported the same logic into our own environment detection code, and it works as expected.
